org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/commands/IExecutableCommandService.java

```diff
@@ -29,11 +28,11 @@
   /**
    * Called on server initialize, only if the client supported dynamicRegistration of commands.
    */
-  default void initializeDynamicRegistration(final Function1<? super String, ? extends IDisposable> register) {
+  default void initializeDynamicRegistration(Function1<? super String, ? extends IDisposable> register) {
```

Comment:
Just checking but IIRC `Function1` can't be replaced by `java.util.Function` without breaking backwards compatibility can't it?

Reply:
yes there is no inheriance yet